src/GraphQL/GraphQL.csproj

```diff
@@ -18,13 +18,15 @@
     <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
     <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
     <GenerateAssemblyInformationalVersionAttribute>false</GenerateAssemblyInformationalVersionAttribute>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
   </PropertyGroup>
 
   <ItemGroup>
     <Compile Include="..\CommonAssemblyInfo.cs" />
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" Condition="$(Configuration) == 'Release'" />
```

Comment:
Is this `Condition` necessary?

Reply:
i try to avoid impacting build perf. and since this interrogates the git files system i would rather only pay that cost in release mode.

Happy to remove if u think it is not worth it

Reply:
It makes sense. What about building in Release configuration not on the CI server? Will it slow down dev machine?

Reply:
> What about building in Release configuration not on the CI server

yep they will pay the price

TBH i never measured this one specifically. I just dont do unnecessary stuff in Debug.

Reply:
I used to run benchmarks in release configuration. This is a rather painstaking job with many edits and rebuilds. I worked on graphql-dotnet/parser#53 recently and after merging this I am going to work on this repo. So these changes may affect me. Could you measure the delay build time?